tests/linearGaussian_snpe_test.py

```diff
@@ -329,13 +329,7 @@ def test_api_snpe_c_posterior_correction(sample_with, mcmc_method, prior_str, se
     simulator, prior = prepare_for_sbi(
         lambda theta: linear_gaussian(theta, likelihood_shift, likelihood_cov), prior
     )
-    inference = SNPE_C(
-        prior,
-        simulation_batch_size=50,
-        sample_with=sample_with,
-        mcmc_method=mcmc_method,
-        show_progress_bars=False,
-    )
+    inference = SNPE_C(prior, show_progress_bars=False)
```

Comment:
this change is independent from the rest of the PR. The test still followed the API from sbi<0.14.0